engine/class_modules/priest/sc_priest.hpp

```diff
@@ -728,6 +729,10 @@ struct priest_action_t : public Base, public parse_buff_effects_t<priest_td_t>
     parse_buff_effects( p().buffs.words_of_the_pious );       // Spell Direct amount for Smite and Holy Nova
     parse_buff_effects( p().buffs.gathering_shadows, true );  // Spell Direct amount for Mind Sear (NOT DP)
     parse_buff_effects( p().buffs.devoured_pride );           // Spell Direct and Periodic amount
+
+    // Discipline
+    parse_buff_effects( p().buffs.shadow_covenant );
+    parse_buff_effects( p().buffs.sins_of_the_many, false, true );
```

Comment:
Can you add a comment like the spells above for what exactly this is "automagically" applying?

Reply:
Done